runtime/nls/j9cl/j9jcl.nls

```diff
@@ -570,6 +570,16 @@ J9NLS_JCL_CRIU_FAILED_TO_ENABLE_ALL_RESTORE_OPTIONS.system_action=
 J9NLS_JCL_CRIU_FAILED_TO_ENABLE_ALL_RESTORE_OPTIONS.user_response=
 # END NON-TRANSLATABLE
 
+# This fatal error can only occur if the user attempts to set -Dcom.ibm.fips.mode to a value other than
+# "140-3" if the FIPS 140-3 preview is enabled, or to "140-2" if not
+J9NLS_JCL_FIPS_MODE_SET_TO_INVALID_VALUE=com.ibm.fips.mode cannot be set to value %s
+J9NLS_JCL_FIPS_MODE_SET_TO_INVALID_VALUE.sample_input_1=140-2
+# START NON-TRANSLATABLE
+J9NLS_JCL_FIPS_MODE_SET_TO_INVALID_VALUE.explanation=-Dcom.ibm.fips.mode= used with an inappropriate value (only 140-2 can be used if preview disabled, 140-3 if enabled).
+J9NLS_JCL_FIPS_MODE_SET_TO_INVALID_VALUE.system_action=The JVM will fail to start.
+J9NLS_JCL_FIPS_MODE_SET_TO_INVALID_VALUE.user_response=Check the fips mode value is consistent with the FIPS 140-3 preview being enabled / disabled.
+# END NON-TRANSLATABLE
```

Comment:
Is this added in the middle of the NLS file to match its previous position in IBM J9?

Reply:
Correct. The two new messages at the end were only added Friday and haven't "shipped".